virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc

```diff
@@ -13,7 +13,7 @@ You can create and delete virtual machine (VM) snapshots for VMs, whether the VM
 
 [IMPORTANT]
 ====
-Online snapshots are not supported for virtual machines that have hot-plugged virtual disks.
+Online snapshots are only supported for virtual machines that have hot-plugged virtual disks if the hot-plugged disk is listed in the virtual machine specification.
```

Comment:
I believe its not so clear, because the online snapshot will still be 'supported' and will complete in compare to before, it is just that a hotplug disk which is not persistent i,e not "listed in the virtual machine specification" will simply not be in the snapshot - volumesnapshot will not be created for it.

Reply:
Thanks @ShellyKa13 - How's this:

_Online snapshots are supported for virtual machines that have hot-plugged virtual disks. Add your hot-plugged virtual disk to your virtual machine specification to ensure it is included in the snapshot._

Reply:
It is better although, if one already have the hotplugs in the vm and it has some that are not in the vm spec Im not sure a recommendation to add them to the spec is called for here, they probably know that they added those disks as not persistent, hence they shouldn't be in the snapshot. I think something of a "notice" that those wont be in the snapshot is more accurate - something similar with the unsnapshotable volumes that are in the vm. like "note that hotplugs volumes which are not in the vm spec will not be included in the snapshot" (this also helps infer that if one do want the hotplug to be in the snapshot it should make sure it is in the spec).

Reply:
OK, revising to:

_Online snapshots are supported for virtual machines that have hot-plugged virtual disks. However, hot-plugged disks that are not in the virtual machine specification are not included in the snapshot._

Reply:
I dont think I have lgtm privileges here but
/lgtm